pkg/report/linux_test.go

```diff
@@ -230,3 +235,187 @@ func TestLinuxSymbolizeLine(t *testing.T) {
 		})
 	}
 }
+
+func prepareLinuxReporter(platform string, t *testing.T) *linux {
```

Comment:
`t *testing.T` commonly goes as the first argument.
We commonly refer to `platform` as `arch` throughout the code base, please rename.